vllm/worker/habana_model_runner.py

```diff
@@ -971,11 +992,16 @@ def warmup_graphs(self, strategy, buckets, is_prompt, kv_caches, available_mem):
             self.graphed_buckets.add((batch_size, seq_len, is_prompt))
             self.log_warmup(phase, idx, num_candidates, batch_size, seq_len)
             with HabanaMemoryProfiler() as mem_prof:
-                self.warmup_scenario(batch_size, seq_len, is_prompt, kv_caches)
+                try:
+                    self.warmup_scenario(batch_size, seq_len, is_prompt, kv_caches)
+                except:
+                    print(f"Failed on graph scenario {idx}: batch_size={batch_size}, seq_len={seq_len}, is_prompt={is_prompt}")
```

Comment:
same comment as previously - warmup should not fail, and if it does, we should not suppress those failures

Reply:
Removed